docs/helpers/custom_helpers.rb

```diff
@@ -0,0 +1,48 @@
+require 'yaml'
+
+module CustomHelpers
+
+  def create_navigation
+  	navigation = ""
+  	topic_map = YAML.load_file('build/_topic_map.yml')
+  	topic_map["Topics"].each_with_index { |topic, index|
+  		if topic["Dir"].nil?
```

Comment:
Le'ts extract `topics["Dir"]` and similar entities into some variable. Looks like used in multiple places